### PR TITLE
* OpenSubtitle fetcher added

### PR DESCRIPTION
**Pull request by FurkanSalih**
- Configurations added to Config.ini for OpenSubtitle
- main.py modified

### CODE REVIEW

**Comment by furkanvarol** [comment]
looks good, needs to be merged asap

File: fetchers/opensubtitle.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+      if data == False:
+        return
+      link = str(data[0]['SubDownloadLink'])
+      self.downloadSubtitleFromLink(link, path, filename)
+
+    value = ops.logout()
+
+  def downloadSubtitles(self, path, subLang):
+    dict = {}
+    os.chdir(path)""
```
Comment (kubudi) [inline fetchers/opensubtitle.py:47]: what does this quotes stands for?

**Comment by kubudi** [comment]
@FurkanSalih as soon as we figure these mysterious quotes, it's good to merge

**Comment by kubudi** [comment]
Merged manually

**Pull request closed without merging**